examples/neural_doodle.py

```diff
@@ -52,6 +52,11 @@
 from keras.preprocessing.image import load_img, img_to_array
 from keras.applications import vgg19
 
+try:
+    xrange          # Python 2
+except NameError:
+    xrange = range  # Python 3

```

Comment:
Instead, I suggest that you replace the calls to `xrange` with `range` since that is probably what we want here.